admin/acceptance_tests/interface_test.py

```diff
@@ -0,0 +1,134 @@
+# pylint: disable=no-self-use
+
+import pytest
+import re
+
+from . import AbstractViewsTests
+
+
+@pytest.fixture(scope='class')
+@pytest.mark.usefixtures('dbsession')
+def interface_test_data(dbsession):
+
+    from c2cgeoportal_commons.models.main import Interface, Theme, OGCServer, LayerWMS
+
+    dbsession.begin_nested()
+
+    themes = []
+    for i in range(0, 5):
+        theme = Theme(name='theme_{}'.format(i),
+                      ordering=1)
+        dbsession.add(theme)
+        themes.append(theme)
```

Comment:
Wonder if dbsession.add(theme) if usefull as dbsession.add(interface) may do the job (not sure)

Reply:
Yes it should, when you add an instance state to the session, related instances are added too.